dapps/digital-twin-detail/src/components/DigitalTwinDetail.vue

```diff
@@ -183,4 +183,17 @@ export default DigitalTwinDetailComponent;
   margin-left: auto;
   margin-right: auto;
 }
+
+/deep/ .content-card {
+  background: white;
+  border-radius: 4px;
+  width: 564px;
+  padding: 32px 24px;
```

Comment:
```suggestion
  padding: 24px 24px;
```
new standard in the concept